easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,38 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'c-ares'
+version = '1.18.1'
+
+homepage = 'https://github.com/c-ares/c-ares'
+description = """This is c-ares, an asynchronous resolver library. 
+It is intended for applications which need to perform DNS queries 
+without blocking, or need to perform multiple DNS queries in parallel. 
+The primary examples of such applications are servers which communicate 
+with multiple clients and programs with graphical user interfaces."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/c-ares/c-ares/archive']
+sources = ['cares-%s.tar.gz' % version.replace('.', '_')]
+
+checksums = [
+    '414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9',  # cares-1_18_1.tar.gz
+]
```

Comment:
This can/should be trimmed since there's only a single source file:

```python
source_urls = ['https://github.com/c-ares/c-ares/archive']
sources = ['cares-%s.tar.gz' % version.replace('.', '_')]
checksums = ['414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9']